compiler/parser-lossless/src/lib.rs

```diff
@@ -162,8 +162,8 @@ pub enum ExpressionKind {
     MethodCall,
     Parenthesized,
     Repeat,
-    // program.id, block.height, etc
-    SpecialAccess,
+    Intrinsic,
+    SpecialAccess, // TODO: fold into Intrinsic
```

Comment:
Let's make sure there's a separate issue tracking this TODO so we don't forget about it.